website/project/views/node.py

```diff
@@ -1068,7 +1070,11 @@ def search_node(**kwargs):
     auth = kwargs['auth']
     node = Node.load(request.json.get('nodeId'))
     include_public = request.json.get('includePublic')
+    size = float(request.json.get('size', '5').strip())
```

Comment:
Also, if the input is invalid, e.g. the provided `size` is not an int, we should return a proper error response.